src/_includes/site-header.html

```diff
@@ -1,7 +1,10 @@
 <header class="site-header">
   {% if page.url == "/" %}
-  <h1 class="site-name">{{ siteData.name }}</h1>
+  <h1 class="site-name top">{{ siteData.name }}</h1>
```

Comment:
Does having both of these class names help in some way? Or could you style `.site-name` with the styles you've added to `.top`? If it's to distinguish this from the site name in the footer, consider using the existing structure to make that more clear; instead of adding a class name here, you could use `.site-header .site-name` as a selector for the styles that should _only_ be on the header version of the site name.